doc/usage/extensions/intersphinx.rst

```diff
@@ -148,3 +148,14 @@ project. The following example prints the Intersphinx mapping of the Python 3
 documentation::
 
    $ python -msphinx.ext.intersphinx https://docs.python.org/3/objects.inv
+
+Using Intersphinx with inventory file under login
```

Comment:
I prefer to use "Basic Authentication" in here instead of "login". What do you think?

Reply:
Will the sentence be "Using Intersphinx with the inventory file under Basic Authentication"?

Reply:
Yes, I feel "under login" is a bit ambiguous because we have many login methods. This only describes about Basic Authentication. So it is better to mention to concrete login method in title.

Reply:
Done!